### PR TITLE
* MDF [doc] update docs, add parquet section

### PR DESCRIPTION
**Pull request by JaylinYu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,11 +10,11 @@
 [![codecov](https://codecov.io/gh/emqx/nanomq/branch/master/graph/badge.svg?token=24E9Q3C0M0)](https://codecov.io/gh/emqx/nanomq)
 [![License](https://img.shields.io/github/license/emqx/nanomq.svg?logoColor=silver&logo=open-source-initiative&label=&color=blue)](https://github.com/emqx/nanomq/blob/master/LICENSE.txt)
 
-NanoMQ MQTT Broker (NanoMQ) is a lightweight and blazing-fast MQTT Broker for the IoT Edge platform. 
+NanoMQ MQTT Broker (NanoMQ) is an ALl-round Edge Messaging platfor, include a lightweight and blazing-fast MQTT Broker for the IoT/IIoT and Messaging Bus for SDV.
```
Comment (lee-emqx) [inline README.md:13]: platform?

File: docs/zh_CN/rule/web-hook.md
Comment (lee-emqx) [inline docs/zh_CN/rule/web-hook.md:1]: Delete this file?

File: README.md (hunk 1 above)
Comment (wanghaEMQ) [reply README.md:13]: a typo: ALl

**Review by wanghaEMQ — COMMENTED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (JaylinYu) [reply README.md:13]: oh shit

File: docs/zh_CN/rule/web-hook.md
Comment (JaylinYu) [reply docs/zh_CN/rule/web-hook.md:1]: yes

**Pull request merged**